vllm/scripts.py

```diff
@@ -82,6 +96,151 @@ def chat(system_prompt: Optional[str], model_name: str,
         print(output)
 
 
+def download_weights(
```

Comment:
I wonder if moving all these functions to a file that's not going to be changed by upstream vllm will help in future merge conflicts